plottable.css

```diff
@@ -78,6 +78,14 @@ svg.plottable {
   fill: #32313F;
 }
 
+.plottable .off-bar-label text {
+  fill: #5279c7;
```

Comment:
This should probably be `#32313F`, which is the color used for the `.dark-label` class, as well as text on `Label`, `Legend`, and `Axis`.